packages/rancher-gatekeeper/generated-changes/patch/Chart.yaml.patch

```diff
@@ -5,7 +5,7 @@
 +  catalog.cattle.io/auto-install: rancher-gatekeeper-crd=match
 +  catalog.cattle.io/certified: rancher
 +  catalog.cattle.io/display-name: OPA Gatekeeper
-+  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.27.0-0'
++  catalog.cattle.io/kube-version: '>= 1.20.0-0'
```

Comment:
Can we do this like '>= 1.20.0-0 < 1.28.0-0'. I can see a case where this can be easily missed. 

- Suppose release Rancher 2.8.1 and later another version 2.8.4 which supports k8s 1.28 but no new chart is being introduced. then updating this chart would fail by the customer since this chart doesn't support 1.28.

Reply:
I have also udpated the branching document